### ISSUE TITLE
Support for peru.yml

### ISSUE THREAD

**Issue by nikolay** [buildinspace/peru#91]
It's annoying if the entire project has adopted `.yml` to have a single black sheep (read `peru.yaml`) using `.yaml`. Maybe a fallback to `peru.yml` should be added if `peru.yaml` does not exist.

**Comment by oconnor663** [comment]
I'm a little hesitant to add in fallback magic, because of the potential for confusion. (Say you're using `peru.yml`, and then somehow `peru.yaml` gets created empty, and now your sync stops working and you have no idea why.) There is the `PERU_FILE_NAME` env var, which you could use as a workaround. Does that sound like it would help your case? We'll probably have a command line flag eventually that does the same thing, so let me know if that would be more convenient.

The recursive repos feature (which is live in `master` but not really announced yet) includes a `peru file` field for modules to specify non-standard filename for their peru metadata. So if there are other projects that are going to depend on yours, that's what they would use.

**Comment by nikolay** [comment]
Well, nobody is gonna use `PERU_FILE_NAME=peru.yml peru ...`, honestly, and because potentially thing may differ from one project to another, setting this globally in the profile may not make sense either.

**Comment by oconnor663** [comment]
@olson-sean-k and I are discussing some related stuff one a [big long diff](https://phabricator.buildinspace.com/D197). I'll get back to this after we figure out what we're going to do there.

**Comment by oconnor663** [comment]
@olson-sean-k and I talked about this today, and we decided not to support it. Our biggest worry is that by supporting it, we'd put a burden on any tool that wants to be peru-aware to know all the different filenames peru might be using.

Agreed that mixing extensions sucks. (We have some of the same with `.travis.yml`.) We went with `.yaml` because that seems to be the Official Extension [suggested by the YAML designers](http://www.yaml.org/faq.html).
